spec/models/plant_accession_spec.rb

```diff
@@ -5,7 +5,7 @@
     it 'will query by permitted params' do
       pas = create_list(:plant_accession, 2)
       filtered = PlantAccession.filter(
-        query: { id: pas[0].id }
+        query: { 'id' => pas[0].id }
```

Comment:
I decided to turn all query parameters to string - that was discussed with Tomek as well. It makes things a little bit cleaner. With strings, however, you can't use the new notations (which is suitable for symbols).